src/extension/commands/client_init.c

```diff
@@ -91,13 +63,16 @@ static dd_result _pack_command(
     dd_mpack_write_nullable_cstr(w, ZSTR_VAL(get_DD_ENV()));
 
     dd_mpack_write_lstr(w, "tracer_version");
-    dd_mpack_write_nullable_cstr(w, NULL);
+    //@todo deharcode this
+    dd_mpack_write_nullable_cstr(w, PHP_DDAPPSEC_VERSION);
 
     dd_mpack_write_lstr(w, "app_version");
-    dd_mpack_write_nullable_cstr(w, NULL);
+    //@todo deharcode this
+    dd_mpack_write_nullable_cstr(w, "0.0");
```

Comment:
This will be handle as part of https://datadoghq.atlassian.net/browse/APPSEC-7305

Reply:
Done